src/drc/api/views/bestandsdeel.py

```diff
@@ -17,8 +20,37 @@
         summary=_("Upload een bestandsdeel."),
     ),
 )
+@extend_schema(
+    responses={
+        200: inline_serializer(
+            name="CustomResponseForWriteOnlyFields",
+            fields={
+                "url": serializers.HyperlinkedIdentityField(
+                    view_name="bestandsdeel_update"
+                ),
+                "lock": serializers.CharField(
+                    help_text=BestandsDeelSerializer._declared_fields["lock"].help_text,
```

Comment:
Kan deze ook niet uit de `Meta` gehaald worden?

Reply:
hmm volgens mij niet want deze word niet gedefineerd in de Meta